ContactsManagerBackend/ContactsManager/ContactsManagerBL/PhonesManager .cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Collections.Generic;
+using AutoMapper;
+using ContactsManagerBL.Models;
+using ContactsManagerDAL;
+using PhoneDAL = ContactsManagerDAL.Models.Phone;
+
+
+namespace ContactsManagerBL
+{
+    public class PhonesManager
+    {
+        private PhonesProvider PhonesProvider { get; } = new PhonesProvider();
+
+        public List<Phone> GetPhones(Guid contactId) => Mapper.Map<List<Phone>>(PhonesProvider.GetPhones(contactId));
+
+        public Phone GetPhone(Guid contactId, Guid Id) => Mapper.Map<Phone>(PhonesProvider.GetPhone(contactId, Id));
+
+        public Phone CreatePhone(Guid contactId, Phone phone)
+        {
+            if (
+                phone == null ||
+                phone.Number.Number == null
```

Comment:
It would be ok to one-line this bool expression.
Also, phone?.Number?.Number == null is shorter, and checks whether phone.Number is null, which you should do.
This comment applies to other places with similar code as well